app/gui2/src/components/GraphEditor.vue

```diff
@@ -51,6 +54,29 @@ const interactionBindingsHandler = interactionBindings.handler({
   click: (e) => (e instanceof MouseEvent ? interaction.handleClick(e) : false),
 })
 
+// This is where the component browser should be placed when it is opened.
+function targetComponentBrowserPosition() {
+  const editedInfo = graphStore.editedNodeInfo
+  const isEditingNode = editedInfo != null
+  const hasNodeSelected = nodeSelection.selected.size > 0
+  const nodeSize = new Vec2(0, 24)
+  if (isEditingNode) {
+    const targetNode = graphStore.db.nodes.get(editedInfo.id)
+    const targetPos = targetNode?.position ?? Vec2.Zero
+    return targetPos.add(COMPONENT_BROWSER_TO_NODE_OFFSET)
+  } else if (hasNodeSelected) {
+    const gapBetweenNodes = 48.0
```

Comment:
Another thing that definitely should be a constant. Also, why isn't the default gap value used instead?